src/shared/services/bolt/handleBoltWorkerMessage.ts

```diff
@@ -136,7 +136,13 @@ export const handleBoltWorkerMessage =
           runningCypherQuery = false
           execCloseConnectionQueue()
           postMessage(
-            maybeCypherErrorMessage({ code: err.code, message: err.message })
+            maybeCypherErrorMessage({
```

Comment:
this is a little different to the approach in UPX

there we are capturing the bolt protocol version inside our driver adapter, which we can then refer to when we catch an exception

here, I've found that this handler seems to a few layers of abstraction away from the driver, so I was unsuccessful in getting a reference to the protocol, much less the driver, when an exception occurs

so, I'm just sending back the GQL fields with the usual fields and doing the protocol check in the app layer